src/cwhy/cwhy.py

```diff
@@ -78,7 +78,7 @@ def read_lines(file_path: str, start_line: int, end_line: int) -> (str, int):
         # return the requested lines as a list
         return ("\n".join(lines[start_line:end_line]) + "\n", start_line, end_line)
     except FileNotFoundError:
-        print(f"Cwhy warning: file not found: {file_path.lstrip()}")
+        # print(f"Cwhy warning: file not found: {file_path.lstrip()}")
```

Comment:
Any reason to remove this warning? I've found it helpful for C/C++. Does it come up a lot in Java/Python?

Reply:
I may have eliminated it by changing the regex for Python (which was including internals like `<string>`). OK to restore it. (Why is it useful, though?)

Reply:
When running in non-wrapper mode, non-found files are relatively common. I guess it's just good to know that the source of that file file will not be included in the prompt.

Also for reference, I think we are not properly sending nothing in the prompt when a file is not found, and we may include some garbage, will look into that.

Reply:
I also suspect there is an off-by-one error in the reported line numbers for code snippets.

Reply:
FYI, both issues now fixed in 61e374f4c8563ccdd6a300cc4c21725ab7fe9987.